includes/class-sensei-course.php

```diff
@@ -205,18 +312,40 @@ public function add_custom_navigation() {
 			return;
 		}
 
+		$tabs       = $this->get_course_custom_navigation_tabs();
+		$screen_ids = wp_list_pluck( $tabs, 'screen_id' );
+
+		// Exclude admin pages (admin.php) because `in_admin_header` hook
+		// doesn't work properly for this case.
+		$screen_ids = array_filter( $screen_ids, [ $this, 'not_admin_page' ] );
+
+		if ( in_array( $screen->id, $screen_ids, true ) && 'term' !== $screen->base ) {
+			$this->display_courses_navigation( $screen, $tabs );
+		}
+	}
+
+	/**
+	 * Get course custom navigation tabs.
+	 *
+	 * @return array Array of tabs.
+	 */
+	private function get_course_custom_navigation_tabs() {
 		$tabs = [
-			[
+			'all-courses'       => [
 				'label'     => __( 'All Courses', 'sensei-lms' ),
 				'url'       => admin_url( 'edit.php?post_type=course' ),
 				'screen_id' => 'edit-course',
 			],
-
-			[
+			'course-categories' => [
 				'label'     => __( 'Course Categories', 'sensei-lms' ),
 				'url'       => admin_url( 'edit-tags.php?taxonomy=course-category&post_type=course' ),
 				'screen_id' => 'edit-course-category',
 			],
+			'showcase-courses'  => [
```

Comment:
As we talked in the meeting, we transformed it to a keyed array, so we can manipulate it easily through Sensei Pro.

Reply:
hmmm..What about maybe adding a `order` parameter and ordering the view by that? I think PHP ordering might be fine, but I'm not 100% sure. :thinking:

Reply:
I think it's fine! I found this StackOverflow with some links to the PHP documentation explaining that it's ordered because arrays are an ordered map in PHP. :)
Anyway, if you find any problem with this, let me know that we can fix it.

Reply:
Fair, thanks for the confirmation. :)